project/src/test/java/com/google/sps/SearchDataServletTest.java

```diff
@@ -25,27 +42,86 @@
   private HttpServletResponse secondPostResponse;
   private HttpServletRequest getRequest;
   private HttpServletResponse getResponse;
-  private StringWriter stringWriter;
-  private PrintWriter printWriter;
+  private StringWriter getStringWriter;
```

Comment:
I don't entirely understand why there are multiple StringWriters and PrintWriters - I don't really see where they're used

Reply:
Ah so in the EventCreationServlet, the Servlet calls getWriter() so if I dont use these writers getWriter() will return null.